src/components/viewsComponents/events/constants/EventsData.ts

```diff
@@ -1,56 +1,56 @@
-import { addDays, setHours, setMinutes } from 'date-fns';
+import { addDays, setHours, startOfDay } from 'date-fns';
```

Comment:
In this file you can't remove setting minutes and hours because compare main and on your branch for view events